tonic/src/transport/server/mod.rs

```diff
@@ -897,6 +897,7 @@ impl<L> Router<L> {
     }
 
     /// Create a tower service out of a router.
+    #[deprecated(since = "0.12.4", note = "compose the layers and the `Routes`")]
```

Comment:
So how would you spell this in 0.13?

Reply:
What is the purpose? Should this api be kept in 0.13?

Reply:
I'm asking what the replacement looks like, in order to better understand how much boilerplate this call is hiding.

Reply:
I see. In the case where this API is used, users can build `Routes` in advance, so they can set some layers to the `Routes` with some means, for example `tower::ServiceBuilder`.

Reply:
I see. In the case where this API is used, users can build `Routes` in advance, so they can set some layers to the `Routes` with some means, for example `tower`'s `ServiceBuilder`.

Reply:
I understand -- do you have a code snippet showing what that would look like?